### PR TITLE
matching bracket problem solved and readme updated

### PR DESCRIPTION
**Pull request by wordhou**
- [X] Top-level README “Table of Contents” is updated
 - [X] Feature tasks for this challenge are completed
 - [X] Unit tests written and passing
     - [X] “Happy Path” - Expected outcome
     - [X] Expected failure
     - [X] Edge Case (if applicable/obvious)
 - [X] README for this challenge is complete
     - [X] Summary, Description, Approach & Efficiency, Solution
     - [X] Link to code
     - [X] Picture of whiteboard

### CODE REVIEW

File: challenges/lib/src/test/java/challenges/utilities/MultiBracketValidationTest.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.*;
+import static challenges.utilities.MultiBracketValidation.multiBracketValidation;
+
+public class MultiBracketValidationTest {
```
Comment (Mattpet26) [inline challenges/lib/src/test/java/challenges/utilities/MultiBracketValidationTest.java:11]: As always, fantastic work with your tests.

File: challenges/lib/src/main/java/challenges/utilities/MultiBracketValidation.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+
+import challenges.stacksQueues.Stack;
+import challenges.stacksQueues.LinkedListStack;
+
+import java.util.Map;
+import static java.util.Map.entry;
+
+public class MultiBracketValidation {
+    static Map<Character, Character> pairs = Map.ofEntries(
```
Comment (Mattpet26) [inline challenges/lib/src/main/java/challenges/utilities/MultiBracketValidation.java:11]: Nice work using a HashMap with the challenge!
